src/app/translations.tsv

```diff
@@ -48,11 +48,11 @@
 "cover_document"	"Only present on certain resource"	"Présente uniquement sur certaines ressources"
 "dataset_characteristics"	"Characteristics"	"Caractéristiques"
 "export"	"Export as %{type}"	"Exporter en %{type}"
-"settings"	"Settings"	"Paramètres"
+"settings"	"Clear"	"Effacer"
```

Comment:
Why is `settings` translated as `Clear` ?
Please use the `clear` translation and delete this one if it's no used please.